jupyter_server/extension/serverextension.py

```diff
@@ -45,7 +45,7 @@ def _get_config_dir(user=False, sys_prefix=False):
     return extdir
 
 
-def _get_extmanager_for_context(user=False, sys_prefix=False):
+def _get_extmanager_for_context(write_dir="jupyter_server_config.d", user=False, sys_prefix=False):
```

Comment:
Sorry for commenting on an old, merged PR, but what is this argument for? It is unused and undocumented in docstring.

Reply:
Good catch, @vidartf. I've updated this function to use the argument and added a docstring in #297.